en/docs/get-started/quick-start-guide.md

```diff
@@ -79,7 +83,7 @@ Follow this procedure to further develop the API by adding a new resource and si
      | **Return Type** | `string|error` |
```

Comment:
Need to update some of the field names to align with the new names. For example, 
change **Path** to **Resource Path**
With regard to query parameter field names, change **Name** to **Param Name**

Reply:
Also might have to change the Return Type value to `string`. specifying as `string|error` results in an error later when adding the return expression.

Reply:
Updated the parameter name.

It works with `string|error` as the return type as agreed offline.